src/RouteParser.php

```diff
@@ -29,8 +29,8 @@
      * Here one route string was converted into two route data arrays.
      *
      * @param string $route Route string to parse
-     * 
-     * @return mixed[][] Array of route data arrays
+     *
+     * @return (mixed[])[] Array of route data arrays
```

Comment:
That line was my entry point into going over the psalm output; It looks like with the other changes this one was moot.